### PR TITLE
Get all activity

### PR DESCRIPTION
**Pull request by sworlan1**
Displaying progress statements based on activity

### CODE REVIEW

File: server/api/stats_api.py [hunk 1]
```diff
@@ -118,3 +118,32 @@ def get(self, project_id):
...
+            404:
+                description: No activity yet
+            500:
+                description: Internal Server Error
+        """
+        try:
+            all_user_stats = StatsService.get_all_user_stats()
+            return all_user_stats.to_primitive(), 200
+        except NotFound:
+            return {"Error": "No activity yet"}, 404
```
Comment (sworlan1) [inline server/api/stats_api.py]:
```suggestion (thinking about gamification using activity tracking"
             progress_level = 0
             print "No Progress"
             return {"Error": "No activity yet"}, 404
```

**Comment by xamanu** [comment]
@sworlan1, @ethan-nelson this is code quite out of sync from 2017. Shall we try to bring this up again?

**Comment by ethan-nelson** [comment]
Yeah this is majorly out of date. My philosophy on the architecture has changed a bit since this time, too, so I may just scrap the service file part and start fresh.

**Comment by xamanu** [comment]
Moving back into the deep spheres of development. Please file a PR once it is ready. Thanks.

**Pull request closed without merging**